homework/lesson_number_2/lesson_number_2/Program.cs

```diff
@@ -0,0 +1,214 @@
+﻿using System;
+using System.Linq.Expressions;
+
+namespace lesson_number_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] numbers = {5, 3, 1, 4, 2, 12, 15, 9, 8};
+
+            int[] a = SortArray(numbers);
+            int[] b = AddElementEndOfArray(numbers, 6);
+            int[] c = AddElementStartOfArray(numbers, 6);
+            int[] d = AddElementAnyPositionInArray(numbers, 6, 5);
+            int[] e = RemoveElementStartOfArray(numbers);
+            int[] f = RemoveElementEndOfArray(numbers);
+            int[] g = RemoveElementAnyPositionInArray(numbers, 2);
+          
+            //change the parameter below to show the outputs of the various array methods
+            PrintUpdatedArray(a);
```

Comment:
Even better, you could submit a main that tests out each of the methods and prints a message before each one.